controllers/datadogagent/feature/eventcollection/feature.go

```diff
@@ -54,6 +74,12 @@ func (f *eventCollectionFeature) Configure(dda *v2alpha1.DatadogAgent) (reqComp
 		f.serviceAccountName = v2alpha1.GetClusterAgentServiceAccount(dda)
 		f.rbacSuffix = common.ClusterAgentSuffix
 
+		if apiutils.BoolValue(dda.Spec.Features.EventCollection.UnbundleEvents) && len(dda.Spec.Features.EventCollection.CollectedUnbundledEvents) > 0 {
+			f.configMapName = apicommonv1.GetConfName(dda, nil, apicommon.DefaultKubeAPIServerConf)
+			f.unbundleEvents = *dda.Spec.Features.EventCollection.UnbundleEvents
+			f.unbundleEventTypes = dda.Spec.Features.EventCollection.CollectedUnbundledEvents
+		}
```

Comment:
should we log a warning if `UnbundleEvents` is set to `true` but for some reason `CollectedUnbundledEvents == 0` ?